index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Trek!</title>
+
+    <link rel="stylesheet" href="styles/foundation.css" />
+    <link rel="stylesheet" href="styles/normalize.css" />
+    <link href="styles/main.css" rel="stylesheet" />
+    <script src="js/vendor/modernizr.js"></script>
+  </head>
+  <body>
+
+    <div id='top-show' class='flexbox-container'>
+      <div class='row'>
+        <div id="head" class="small-3 small-centered columns text-center flexbox-container">
```

Comment:
I'm seeing lots of `<div>`s here where a tag with more semantic weight might be appropriate. A screen reader needs semantic tags regardless of whether the content is dynamically generated.